src/views/RasterPlot/TimeScrollView/paint.ts

```diff
@@ -18,24 +18,29 @@ const highlightedRowFillStyle = '#c5e1ff' // TODO: This should be standardized a
 export const paintAxes = <T extends {[key: string]: any}>(context: CanvasRenderingContext2D, props: TSVAxesLayerProps<T> & {'selectedPanelKeys': string[]}) => {
     // I've left the timeRange in the props list since we will probably want to display something with it at some point
     // Q: maybe it'd be better to look at context.canvas.width rather than the width prop?
-    const {width, height, margins, panels, panelHeight, perPanelOffset, selectedPanelKeys, timeTicks} = props
+    const {width, height, margins, panels, panelHeight, perPanelOffset, selectedPanelKeys, timeTicks, hideTimeAxis} = props
     context.clearRect(0, 0, context.canvas.width, context.canvas.height)
     
     // x-axes
-    context.strokeStyle = 'black'
-    drawLine(context, margins.left, height - margins.bottom, width - margins.right, height - margins.bottom)
+    if (!hideTimeAxis) {
+        context.strokeStyle = 'black'
+        drawLine(context, margins.left, height - margins.bottom, width - margins.right, height - margins.bottom)
+    }
 
     // time ticks
     for (let tt of timeTicks) {
         // const frac = (tt.value - timeRange[0]) / (timeRange[1] - timeRange[0])
         // const x = margins.left + frac * (width - margins.left - margins.right)
         context.strokeStyle = tt.major ? 'gray' : 'lightgray'
         drawLine(context, tt.pixelXposition, height - margins.bottom, tt.pixelXposition, margins.top)
```

Comment:
Oh, right, this is the code that draws the vertical separator lines that are visible across the time series.